structure_tests/README.md

```diff
@@ -0,0 +1,53 @@
+
+To use this test image with any cloudbuild, add the following build step to the **end** your container build config (cloudbuild.yaml or cloudbuild.json):
+
+	name: gcr.io/gcp-runtimes/structure_test
+	args:
+		- <your_target_image>
+
+It's **very important that this step appears at the end of your build** (or at least after the image itself it assembled by Docker); without a built image, there will be nothing to test, and your build will fail!
+
+Tests within this framework are specified through a JSON config file, by default called `structure_test.json` (this can be specified through a `--config` flag argument to the build step). This file will be copied into the workspace of the structure test image and loaded in by the test driver, which will execute the tests in order. Within this config file, three distinct types of tests can be written:
+
+- Command Tests (testing output/error of a specific command issued)
+- File Existence Tests (making sure a file is, or isn't, present in the file system of the image)
+- File Content Tests (making sure files in the file system of the image contain, or do not contain, specific contents)
+
+## Command Tests
+Command tests ensure that certain commands run properly on top of the shell of the target image. Regexes can be used to check for expected or excluded strings in both stdout and stderr. Additionally, any number of flags can be passed to the argument as normal.
+
+####Supported JSON Fields:
+
+- Name (string, **required**): The name of the test
+- Command (string, **required**): The command to run
+- Flags (string[], *optional*): Optional flags to pass to the command
+- Expected Output (string[], *optional*): List of regexes that should match the stdout from running the command.
+- Excluded Output (string[], *optional*): List of regexes that should **not** match the stdout from running the command.
+- Expected Error (string[], *optional*): List of regexes that should match the stderr from running the command.
+- Excluded Error (string[], *optional*): List of regexes that should **not** match the stderr from running the command.
+
+
+##File Existence Tests
+File existence tests check to make sure a specific file (or directory) exist within the file system of the image. No contents of the files or directories are checked. These tests can also be used to ensure a file or directory is **not** present in the file system.
+
+####Supported JSON Fields:
+
+- Name (string, **required**): The name of the test
+- Path (string, **required**): Path to the file or directory under test
+- IsDirectory (boolean, **required**): Whether or not the specified path is a directory (as opposed to a file)
+- ShouldExist (boolean, **required**): Whether or not the specified file or directory should exist in the file system
+
+
+##File Content Tests
```

Comment:
added examples directly into the text. if you'd prefer a link instead let me know